wp-content/themes/core/assets/css/src/admin/block-editor/alignments.pcss

```diff
@@ -0,0 +1,77 @@
+
+/* -----------------------------------------------------------------------------
+ *
+ * WP Core Block & Editor Alignment Styles & Overrides
+ *
+ * Styles that apply to WP Core blocks or reset styles applied
+ * by the block editor and wp-admin.
+ *
+ * ----------------------------------------------------------------------------- */
+
+.block-editor-block-list__layout {
+
+	.wp-block-audio,
+	.wp-block-video,
+	.wp-block-embed,
+	figure.wp-block-image {
```

Comment:
@dpellenwood considered doing the same `block-editor.pcss` approach for the `css/theme/media/` and the other more sink base level implementations for consistency. Thoughts?

Reply:
Yes, I think that would be a nice addition as well.